OracleInstantClient/README.md

```diff
@@ -29,6 +29,20 @@ Oracle Database 12.1 or later.  Applications using Oracle Call Interface (OCI)
 19, 18 or 12.2 can connect to Oracle Database 11.2 or later.  Some tools may
 have other restrictions.
 
+## Images
+
+Pre-built images for Instant Client are in the [GitHub Package
+Registry](https://github.com/orgs/oracle/packages).  They are built from the
+the Dockerfiles in this repository.
+
+For example, to pull an Oracle Linux 8 image with Instant Client 21c, execute:
```

Comment:
This is probably better phrased as, "_to pull an  image based on Oracle Linux 8 that already has the Oracle Instant Client 21c installed_" and I would recommend calling out whether or not its based on `oraclelinux:8` which means `dnf` is available or `oraclelinux:8-slim` which only has `microdnf`.

Reply:
I'll do some tweaks. But your suggestion is wordy. And the slim distinction is overkill for this brief reference.